config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json

```diff
@@ -45,13 +45,14 @@
 # ------------------------------------------
 
    "RELEASES"  : {
-                   "0.9." : [
-                              "RobotFramework_TestsuitesManagement change 0.9 New RobotFramework 6.1 adaptation; Maintenance of log output"
-                            ],
                    "0.10." : [
-                               "RobotFramework_TestsuitesManagement change 0.10 Improve dotdict feature",
-                               "RobotFramework_TestsuitesManagement change 0.10 Update logging",
-                               "RobotFramework_TestsuitesManagement change 0.10 Code maintenance"
+"
+* Improve dotdict feature
+* Update logging
+* Code maintenance
+* further stabilization of implementation
+* extension of selftest                 
+"
```

Comment:
Hi Son,

please align the typing and wording:
First character in items capitalized.
Use past instead of imperative: "Updated" instead of "Update".
"Code maintenance" and "further stabilization ..." is redundant in my opinion. This can be shortened.

Reply:
Hello Holger,

This pull-request came from Thomas, so Thomas will update the typing and wording as you mentioned above :)

Thank you,
Son